meta.go

```diff
@@ -0,0 +1,14 @@
+package tokentpr
+
+// Metadata to create operatorkit TPR.
+
+const (
+	// Name represents the name of the third party resource within Kubernetes.
+	Name = "token.giantswarm.io"
```

Comment:
Then lets please also rename the repo. Shouldn't be too bad in this early stage.